packages/wrangler/src/d1/create.tsx

```diff
@@ -81,13 +76,11 @@ export const Handler = withConfig<HandlerOptions>(
 							? ` using primary location hint ${location}`
 							: ``}
 					</Text>
-					{experimentalBackend && (
-						<Text>
-							Created your database using D1&apos;s new storage backend. The new
-							storage backend is not yet recommended for production workloads
-							and does not yet support backups.
-						</Text>
-					)}
+					<Text>
+						Created your database using D1&apos;s new storage backend. The new
+						storage backend is not yet recommended for production workloads, but
+						backs up your data via point-in-time restore.
```

Comment:
Should this message be updated?

Reply:
Eh we can update that once we go from alpha -> beta